microservices/storageApi/helm/storage-api/values.yaml

```diff
@@ -11,7 +11,7 @@ image:
   pullPolicy: Always
 
 tus:
-  image: h3brandon/tusd_py3
+  image: tosse/tusd_py3
```

Comment:
Is this the helm chart that is used in the deploy?
From what I remember, most of brandons helm charts got moved to https://github.com/bcgov/helm-charts

Reply:
https://github.com/bcgov/helm-charts/tree/master/charts/storage-api

Reply:
I see what you mean, I am not sure which one is being used. I have no documentation on the deployment.

Reply:
This storage-api microservice has not been modified in 3 years while the helm-charts version was last modified 2 years ago. So have made the same change to the helm-chart version, and i will have a PR for review momentarily.